src/components/LoadingPlaceholder.vue

```diff
@@ -18,54 +19,20 @@
   - You should have received a copy of the GNU Affero General Public License
   - along with this program. If not, see <http://www.gnu.org/licenses/>.
 -->
-<docs>
-
-Displays a loading placeholder for conversation messages.
-
-The gradient animation is achieved by having two placeholder elements,
-with opposite gradient directions (regular and reverse) displayed on top
-of each other (overlapped with position: absolute) and then fading between
-each other by animating the opacities.
-
-</docs>
```

Comment:
You don't want to add a description to the placeholder animation style?

Reply:
We could add it as a simple comment, but we don't generate documentation, like Vue-library does, and it will be the only one component with it, so I doubt it'll be useful

As a follow-up, when we're going to re-organize repo structure, I'd like to move all reusable components to a separate folder. There it make sense to add a description